src/pages/documentation/identity-for-edge-network/release-notes.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## February 28, 2023
+
+### iOS AEPEdgeIdentity 1.2.0
+
+* Added tvOS support.
+* Identity map no longer accepts identity item with empty string value for id
```

Comment:
item -> items
also use period at the end of the sentence

Reply:
Also add a period at the end of the sentence.